package.json

```diff
@@ -48,12 +48,13 @@
     "express": "^4.17.1",
     "helmet": "7.0.0",
     "http-status-codes": "^1.4.0",
+    "http-terminator": "^3.2.0",
     "mixpanel": "^0.13.0",
     "mongoose": "7.6.7",
     "newrelic": "8.14.0",
```

Comment:
Why don't we upgrade newrelic? (and maybe axios?)

Reply:
upgrading newrelic throws [this](https://github.com/newrelic/node-newrelic/issues/2041) error.

Reply:
axios updated.